tests/integration/integration_test.go

```diff
@@ -214,13 +214,12 @@ func TestStackOutputsJSON(t *testing.T) {
 	e.RunCommand("pulumi", "login", "--cloud-url", e.LocalURL())
 	e.RunCommand("pulumi", "stack", "init", "stack-outs")
 	e.RunCommand("pulumi", "up", "--non-interactive", "--skip-preview")
-	stdout, stderr := e.RunCommand("pulumi", "stack", "output", "--json")
+	stdout, _ := e.RunCommand("pulumi", "stack", "output", "--json")
```

Comment:
Is this change necessary?

Reply:
Unfortunately, for now, it is.  See: he commit message of ed804925fd2edb5cb11e53ac5a50848d07459c9f for the sad justification.

I should be able to undo this shortly, but for now it's the quickest way to unblock the release.